Add entitlements to metadata (#13728)
canonical/lxd: doc/reference/permissions.md

(permissions-reference)=
# Permissions

> When managing user access via {ref}`fine-grained-authorization`, identities are added to groups, and groups are granted entitlements against specific LXD API resources.
> Each LXD API resource has a particular entity type.
> Each entity type has a set of entitlements that can be granted against API resources of that type.
> Below is a description of each entity type, and a list of entitlements that can be granted against entities of that type.

Reviewer: ```suggestion
When managing user access via {ref}`fine-grained-authorization`, you add identities to groups and then grant entitlements against specific LXD API resources to these groups.

Each LXD API resource has a particular entity type, and each entity type has a set of entitlements that can be granted against API resources of that type.

Below is a description of each entity type, and a list of entitlements that can be granted against entities of that type.
```

Reviewer: Breaking this up a bit to make it easier to digest. :)
